docs/source/rpc.rst

```diff
@@ -15,15 +15,12 @@ RPC and RRef Framework
 ----------------------
 
 Before using RPC and distributed autograd primitives, initialization must take
-place. First, a backend over which RPCs can be sent over must be initialized.
-The default (and currently, only available) implementation is the `ProcessGroup`
-backend, and must be initialized with `torch.distributed.init_process_group
-<https://pytorch.org/docs/stable/distributed.html#torch.distributed.init_process_group>`_
-before using other functions. See the `documentation for
-torch.distributed <https://pytorch.org/docs/stable/distributed.html>`_ for
-additional details. Next, to initialize the RPC framework we need to use
+place. To initialize the RPC framework we need to use
 `init_rpc` which would initialize the RPC framework, RRef framework
-and distributed autograd.
+and distributed autograd. As part of this, it will also initialize a default
+`ProcessGroup` `backend
+<https://pytorch.org/docs/stable/distributed.html#torch.distributed.Backend>`_
+that uses gloo for collective communication.
```

Comment:
Basically, I'm trying to express that `init_rpc` runs `dist.init_process_group(backend=GLOO)` behind the scenes, is there a better way to write this?

Reply:
Rewrite this as follows to make it clear that the default is ProcessGroup but other backends be available:

```
By default, this will initialize the `ProcessGroup` backend. The `ProcessGroup` backend internally uses gloo for communication
```